playbook/app/pb_kits/playbook/pb_drawer/_drawer.tsx

```diff
@@ -120,39 +129,54 @@ const Drawer = (props: DrawerProps): React.ReactElement => {
       }
     };
 
-    window.addEventListener('resize', handleResize);
+    window.addEventListener("resize", handleResize);
 
     // Call handler once on mount to set initial state
     handleResize();
 
     return () => {
-      window.removeEventListener('resize', handleResize);
+      window.removeEventListener("resize", handleResize);
     };
   }, [breakpoint]);
 
-  const modalIsOpened = trigger ? triggerOpened : (opened || isBreakpointOpen);
+  const modalIsOpened = trigger ? triggerOpened : opened || isBreakpointOpen;
+
+  const [animationState, setAnimationState] = useState("");
+
+  useEffect(() => {
+    if (modalIsOpened) {
+      setAnimationState("afterOpen");
+    } else if (!modalIsOpened && animationState === "afterOpen") {
+      setAnimationState("beforeClose");
+      setTimeout(() => {
+        setAnimationState("");
+      }, 200); // closeTimeoutMS
```

Comment:
There's a bit of stuttering that shows up. I wonder if this should be tweaked?

https://github.com/user-attachments/assets/9b0df102-c95f-404b-bab8-79c31656f737

Reply:
Yeah I'd say, we either need to fix this animation in this current story, or get a fast-follow story for this to fix it. The kit is still in beta, but I'd rather fix it in this current story.

Reply:
i fixed it check it out https://pr3891.playbook.beta.px.powerapp.cloud/kits/drawer/react

Reply:
Nice, looks a lot smoother on my end (using Firefox)!